<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpDebugFCPDebugLogPlayerConfigDialog</class>
 <widget class="QDialog" name="SpDebugFCPDebugLogPlayerConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game log configuration for debug</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Game log file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="gameLogFileLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="gameLogFileButton">
     <property name="text">
      <string>Search...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>With players log</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Players log directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="playersLogDirectoryLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="playersLogDirectoryhButton">
        <property name="text">
         <string>Search...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="debugButton">
       <property name="text">
        <string>Debug!</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ResetPushButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>gameLogFileLineEdit</tabstop>
  <tabstop>gameLogFileButton</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>playersLogDirectoryLineEdit</tabstop>
  <tabstop>playersLogDirectoryhButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>debugButton</tabstop>
  <tabstop>ResetPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
